Update main.jsx

emilyaudela opened the pull request:
Since the positioning of the action toggle has changed, the text in the above paragraphs is out of context and doesn't make sense:
<img width="729" alt="Screen Shot 2019-07-10 at 11 18 07 AM" src="https://user-images.githubusercontent.com/36699353/60990200-21475e00-a305-11e9-8f95-d566fafa5bcd.png">


1) "This information..." –– What information? Using "this" would imply we had mentioned it previously.
2) "We use other tracking tools..." –– other than what?

 I have made some text changes to make this flow more logically from one paragraph to the next.

emilyaudela committed:
Update main.jsx

Since the positioning of the action toggle has changed, the text in the above paragraphs is out of context and doesn't make sense:

1) "This information..." –– What information? Using "this" would imply we had mentioned it previously.
2) "We use other tracking tools..." –– other than what?

 I have made some text changes to make this flow more logically from one paragraph to the next.

a8ci18n commented on client/me/privacy/main.jsx:
```diff
@@ -84,7 +84,7 @@ const Privacy = createReactClass( {
 							<p>
 								{ translate(
-									'This information helps us improve our products, make marketing to you more ' +
+									'The information you choose to share helps us improve our products, make marketing to you more ' +
```
Hi! I've found a possible matching string that has already been translated **22** times:
`translate( 'This information helps us improve our products, make marketing to you more relevant, personalize your WordPress.com experience, and more as detailed in our {{privacyPolicyLink}}privacy policy{{/privacyPolicyLink}}.' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

a8ci18n commented on client/me/privacy/main.jsx:
```diff
@@ -96,7 +96,7 @@ const Privacy = createReactClass( {
 							<p>
 								{ translate(
-									'We use other tracking tools, including some from third parties. ' +
+									'In addition to our analytics tool, we use other tracking tools, including some from third parties. ' +
```
Hi! I've found a possible matching string that has already been translated **23** times:
`translate( 'We use other tracking tools, including some from third parties. {{cookiePolicyLink}}Read about these{{/cookiePolicyLink}} and how to control them.' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

michelleweber commented:
Howdy! I don't have access to the test version, for a copy review, can you post screenshots here (or better yet, just paste in the text you'd like reviewed)? Thanks!

emilyaudela commented:
Here are the 2 lines I have updated: 

1.
Original String:
This information helps us improve our products, make marketing to you more relevant, personalize your WordPress.com experience, and more as detailed in our privacy policy

Changed to
The information you choose to share helps us improve our products, make marketing to you more relevant, personalize your WordPress.com experience, and more as detailed in our privacy policy.

2.
Original String:
We use other tracking tools, including some from third parties. Read about these and how to control them.

Changed to:
In addition to our analytics tool, we use other tracking tools, including some from third parties. Read about these and how to control them.
